lib/Constants.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
     ComponentRepository : process.env.ADAPT_COMPONENT || 'https://github.com/adaptlearning/adapt-component',
     ComponentRepositoryName : 'adapt-component',
     DefaultBranch : process.env.ADAPT_BRANCH || 'master',
-    Registry: getRegistry(),
-    HomeDirectory : searchForHome()
+    HomeDirectory : searchForHome(),
+    getRegistry:getRegistry
```

Comment:
I don't understand why this is changed from a cached constant to a function. It'll have to do multiple `fs.existsSync` calls rather than only one, as before.

Reply:
I had quite a bit of trouble with this and once it was working it was a relief and I must have just stopped there! I've amended to cache the value.